WebRoot/templates/prefs/Pages.template

```diff
@@ -967,7 +967,7 @@ and the container element is replaced with that control.
 						</tr>
 					<$ } $>
 					<tr><td colspan=2></td></tr>
-					<tr><td style='text-align:left' colspan=2><b><$=ZmMsg.accountFromPrompt$></b></td></tr>
+					<tr><td style='text-align:left' class='ZOptionsSubHeadingLabel' colspan=2><$=ZmMsg.accountFromPrompt$></td></tr>
```

Comment:
Good catch. Actually this is not at all required. Removed here: 170445c09d8435d39607eedcd49019ba84a85152